### DISCUSSION TITLE
Country Map chart - map of Belarus is displayed incorrectly

### DISCUSSION THREAD

**Discussion by CattyRu** [apache/superset#28195] [Q&A / Help] [answered]
Good day. I added the map to the county map chart according to the instructions from the Superset documentation (https://superset.apache.org/docs/miscellaneous/country-map-tools/). I can copy any geojson from the official ones, name it differently and it will work. And I can build my own geojson, but it doesn't always work.
Now I am trying to add a map of Belarus presented in geojsons 'all_region' and 'all_region_0_02' with a full and significantly reduced number of coordinates, respectively. Geojsons are checked using the mapster-right-hand-rule-fixer website for compliance with the right-hand rule (https://mapstertech.github.io/mapster-right-hand-rule-fixer/).
[all_region.json](https://github.com/apache/superset/files/15093277/all_region.json)

[all_region_0_02.json](https://github.com/apache/superset/files/15093279/all_region_0_02.json)
When adding these geojsons completely, a square is displayed instead of the map of Belarus. But if I add them individually, then some areas are displayed correctly.
Displayed correctly:
[Homel_region.json](https://github.com/apache/superset/files/15093333/Homel_region.json)

[Mahilyow_region.json](https://github.com/apache/superset/files/15093335/Mahilyow_region.json)

[Minsk_region.json](https://github.com/apache/superset/files/15093336/Minsk_region.json)

[Vitsebsk_region.json](https://github.com/apache/superset/files/15093337/Vitsebsk_region.json)

Displayed as a square:
[Brest_region.json](https://github.com/apache/superset/files/15093403/Brest_region.json)

[Hrodna_region.json](https://github.com/apache/superset/files/15093404/Hrodna_region.json)

[Minsk.json](https://github.com/apache/superset/files/15093405/Minsk.json)

Images of correct areas. This image can be repeated by adding a map from this json: 
[correct_regions.json](https://github.com/apache/superset/files/15093474/correct_regions.json):
![normal_regions](https://github.com/apache/superset/assets/145757773/8187d653-c294-4b1c-b10c-5fee3ef11cfb)

Images incorrect areas. This image can be repeated with any json except "Displayed correctly":
![square_region](https://github.com/apache/superset/assets/145757773/e7b06fde-8302-4ad3-ac40-356293f3d776)

I really don't understand the difference between the ones shown in "Displayed correctly" and "Displayed as a square". I will be glad of any advice.

**Expected results**
The map of Belarus is displayed in the Country Map

**Actual results**
The map of Belarus is displayed in the Country Map as a square

**How to reproduce the bug**
Add map according to the instructions from the Superset documentation.

**Environment**
- superset version: v3.1.1 
- python version: Python 3.12.1
- node.js version: v16.20.2
- npm version: 8.19.4
- I built superset from source

**Comment by yangxin9003** [comment, marked as answer]
I met the same problem, and i found a [article](https://ruky.me/how-to-correctly-load-your-country-in-to-apache-superset/).
I try the script in the article(step 5). it's worked!
